src/main/java/com/google/sps/data/Post.java

```diff
@@ -131,6 +145,51 @@ public void requestToPost(HttpServletRequest request, String collegeId) {
     Calendar nowTime = Calendar.getInstance(TimeZone.getTimeZone("America/Los_Angeles"));
     year = nowTime.get(Calendar.YEAR);
 
+    // Implements a try-catch statement in case the Servlet is accessed directly, instead of through Blob Upload URL.
+    try {
+      // If there is an image uploaded, set the Blob Key and Serving Url.
+      blobKey = getBlobKey(request, "foodImage");
+      imageServingUrl = getServingUrl(blobKey);
+      if (imageServingUrl == null) {
+        imageServingUrl = "no image";
```

Comment:
`imageServingUrl` name suggests that this should be a URL. Setting it to `no image` contradicts the meaning of the name. So, I would prefer if null was used here instead.

Reply:
Makes sense!